public/js/task-detail.js

```diff
@@ -300,13 +300,31 @@ class TaskDetail {
       return;
     }
 
-    const logsHTML = logs.map(log => `
-      <div class="flex items-start space-x-2 mb-2 text-sm font-mono">
-        <span class="text-gray-400">${new Date(log.timestamp).toLocaleTimeString()}</span>
-        <span class="text-${this.getLogColor(log.level)}-400 font-medium">[${log.level.toUpperCase()}]</span>
-        <span class="text-gray-300 flex-1 whitespace-pre-wrap">${this.escapeHtml(log.message)}</span>
-      </div>
-    `).join('');
+    const logsHTML = logs.map(log => {
+      const timestamp = new Date(log.timestamp);
+      const timeString = timestamp.toLocaleTimeString(undefined, {
+        hour: '2-digit',
+        minute: '2-digit',
+        second: '2-digit'
+      });
+      const fullTimeString = timestamp.toLocaleString(undefined, {
```

Comment:
duckling this and timesstamp and timestring is duplicated a few times. Consolidate